src/chunk_dispatch_state.c

```diff
@@ -143,7 +221,47 @@ chunk_dispatch_exec(CustomScanState *node)
 		MemoryContextSwitchTo(old);
 
 		/* Convert the tuple to the chunk's rowtype, if necessary */
-		tuple = ts_chunk_insert_state_convert_tuple(cis, tuple, &slot);
+#if PG12_LT
+		tuple = ts_chunk_insert_state_convert_tuple(cis, tuple, NULL);
```

Comment:
Is it on purpose address to `slot` is not provided, so it is not set? Seems to be different to the original code.

Reply:
I'll need to double check with PG11, I think I originally had the logic so that `slot` was always `state->slot`, so the store would be redundant, but that may have been on 12.